src/ethereum_test_specs/base.py

```diff
@@ -119,5 +139,51 @@ def get_next_transition_tool_output_path(self) -> str:
             str(current_value),
         )
 
+    def is_slow_test(self) -> bool:
+        """Check if the test is slow."""
+        if self._request is not None and hasattr(self._request, "node"):
+            return self._request.node.get_closest_marker("slow") is not None
+        return False
+
+    def is_negative_test(self) -> bool | None:
```

Comment:
same here, and all the usage elsewhere
```suggestion
    def is_exception_test(self) -> bool | None:
```